plugin/init.lua

```diff
@@ -141,41 +156,53 @@ end
 --- @param data string
 --- @return string
 local function sanitize_json(data)
+	wezterm.emit("resurrect.sanitize_json.start", data)
 	data = data:gsub("[\x00-\x1F\x7F]", function(c)
+		local byte = string.byte(c)
 		if is_windows then
-			local byte = string.byte(c)
 			if byte == 0x0A then
 				return "\n"
 			elseif byte == 0x0D then
 				return "\r"
 			elseif byte == 0x09 then
 				return "\t"
 			else
-				return string.format("\\u%04X", string.byte(c))
+				wezterm.log_info("Invalid control character: " .. byte)
```

Comment:
What is this used for? Just a warning? If so, then i think it should be `wezterm.log_warn()`

Reply:
Yes - I found it useful when I had some issues on Windows to figure out which control character was causing things to be. This can be removed if you'd like.

Reply:
If the return below (line 171) is transformaing invalid control chars to valid, then I think it should be removed.

Reply:
Removed